### DISCUSSION TITLE
Jump to document in file manager?

### DISCUSSION THREAD

**Discussion by istvandbsd** [Zettlr/Zettlr#5516] [Q&A]
Hi,

Apologies if I missed how to do this. I ran through the documentation, and existing GH issues and discussions. 

Got a lot of notes already and being able to highlight the file in the file manager tree would be useful.
The workflow I'm thinking of is very simple: right-click on the open document's tab -> (Highlight|Jump to|Go to) in File Manager -> file is brought to view, recursively opening any parent directories.

Any way to do this currently? Would it be useful to others as well?

**Comment by nathanlesage** [comment]
There is already a mechanism for that. It should be straight forward to implement this.

**Comment by istvandbsd** [comment]
Thanks! Where do we go from here? I see you have a long list of features in the roadmap - I might give it a go if that's okay. I could use some pointers though, links to the mechanism you meant would be great.

**Reply by nathanlesage** [reply]
There was a mechanism "scrollIntoView" somewhere in the file manager if I'm not mistaken. It may have gotten lost in the past year, but it's definitely *somewhere* in the git history. I'm pretty swamped right now so I can't go have a look unfortunately
